### DISCUSSION TITLE
for TDesign Vue-next opinions

### DISCUSSION THREAD

**Discussion by PolarishT** [Tencent/tdesign-vue-next#2298] [General]
Tdesign 非常好用 代码的复用性特别的高 组件之间的扩展性和扩展性也特别的好 是非常值得尝试的开发框架 
但是 有几点问题
1. 文档过于简洁 对许多方法细节不到位 使用者需要去猜测意图
2. 对于poupconfirm的兼容性存在非常大的问题 如使用 tavatar 包裹出现错位问题
3. 只提供了headmenu theme 的修改方法 对content和footer组件没有看到类似的theme 修改方法
未来的希望 
1. 完善文档 
2. 完善方法注释
![图片](https://user-images.githubusercontent.com/72297937/214326622-303b3526-872c-4f76-aec2-737d76d7f584.png)

**Comment by PengYYYYY** [comment]
感谢您的意见
